kensu/utils/kensu_conf_file/conf_file.py

```diff
@@ -0,0 +1,38 @@
+import os
```

Comment:
code exactly the same, just moved to separate file from `kensu/utils/helpers.py`